tests/unit/schema/test.firefox-schemas-import.js

```diff
@@ -37,12 +38,16 @@ async function createZipFile() {
 
   await new Promise((resolve) => {
     zipfile.outputStream
-      .pipe(fs.createWriteStream('mozilla-central.zip'))
+      .pipe(fs.createWriteStream(FAKE_ZIP_FILE_PATH))
       .on('close', () => resolve());
     zipfile.end();
   });
 }
 
+async function deleteZipFile() {
+  fs.unlinkSync(FAKE_ZIP_FILE_PATH);
+}
+
```

Comment:
once we change this patch to use the API promised by "tmp-promise", this may be done using the `cleanup` method described in the "tmp-promise" API docs ([you can take look to the readme file here](https://www.npmjs.com/package/tmp-promise#usage-api-reference))

Reply:
i've used `removeCallback()` can it be that way?